_posts/blog/2022-06-22-2.1.0-metrics-and-tracing.md

```diff
@@ -148,4 +152,30 @@ docker run -d --rm --name jaeger \
 
 8. View traces in Jaeger UI at http://localhost:16686. You can select the service name on the left panel and click `Find Traces` to view the trace information. If everything is working correctly, then you should see something like the image below.
 
-![Jaeger Screenshot](/images/blog/202206_metrics_and_tracing/Jaeger_Screenshot.png)
+<a class="p-3 border rounded d-block" href="{{ site.baseurl }}/images/blog/202206_metrics_and_tracing/Jaeger_Screenshot.png">
+   <img src="{{ site.baseurl }}/images/blog/202206_metrics_and_tracing/Jaeger_Screenshot.png" class="img-fluid rounded" alt="Jaeger Screenshot"/>
+</a>
+
+## Updated Configuration with Newer Versions of OpenTelemetry
+
+The original tracing example above was written for Accumulo 2.1.0 which uses OpenTelemetry 1.19.0. Accumulo 2.1.4 has been updated to use OpenTelemetry 1.48.0, which requires different configuration. When working with Accumulo 2.1.4 or any deployment using OpenTelemetry 1.48.0 or newer, use OTLP (OpenTelemetry Protocol) instead of the direct Jaeger exporter.
+
+Follow steps 1-4 from the [Tracing Example](#tracing-example) section, then use these updated steps in place of steps 5-6:
+
```

Comment:
Kinda late to mention this, if the user was going step-by-step. Should probably mention which specific steps to skip in the initial Note above, before they get this far. Otherwise, the first time you mention to skip steps 5-6 occur after the user has possibly already done them.